src/jdk.incubator.jextract/share/classes/jdk/internal/clang/libclang/CXToken.java

```diff
@@ -31,15 +31,12 @@
 import java.lang.invoke.VarHandle;
 import java.nio.ByteOrder;
 import jdk.incubator.foreign.*;
-import static jdk.incubator.foreign.CLinker.*;
-import static jdk.internal.clang.libclang.CLayouts.C_INT;
-import static jdk.internal.clang.libclang.CLayouts.C_POINTER;
-
+import static jdk.incubator.foreign.ValueLayout.*;
 public class CXToken {
 
     static final MemoryLayout $struct$LAYOUT = MemoryLayout.structLayout(
-        MemoryLayout.sequenceLayout(4, C_INT).withName("int_data"),
-        C_POINTER.withName("ptr_data")
+        MemoryLayout.sequenceLayout(4, JAVA_INT).withName("int_data"),
+        ADDRESS.withName("ptr_data")
     );
```

Comment:
Why these changes? Shouldn't this still be using `C_INT` and `C_POINTER` ?

Reply:
That's generated code. In this patch I am not touching jextract generated code except for copyright header + libName